### DISCUSSION TITLE
Piggy bank should be marked as a reasonable status after the saving target is achieved

### DISCUSSION THREAD

**Discussion by MasterZhang007** [firefly-iii/firefly-iii#7657] [Firefly III] [answered]
I read the description of "Piggy banks" on the official website, but I still have questions about how to use it.

Assume I have a piggy bank called "New Camera" linked to my saving account, the target amount is €300, and I have €300 saved. Today is a good day, I decide to go shopping and buy me the camera. The money (€300) will be removed from the piggy bank, the piggy bank is now without money(€0).

Now the situation is, the money "Saved so far" is €0, and "Left to save" is €300. This is not what I expected. This "New Camera" piggy bank will not be used to save money in the near future, or never be. Maybe it should be marked as "achieved" or "done" or any other reasonable status.

**Comment by JC5** [comment, marked as answer]
But I can't tell intent from my side. If the piggy bank is no longer you can delete it.

**Reply by MasterZhang007** [reply]
Will deleting this piggy bank have side effects on completed transactions?

**Reply by JC5** [reply]
No :)

**Reply by MasterZhang007** [reply]
Great!
